packages/libs/react-ui/src/components/SelectField/SelectField.tsx

```diff
@@ -0,0 +1,27 @@
+import type { IInputWrapperProps } from '@components/InputWrapper';
+import { InputWrapper } from '@components/InputWrapper';
+import type { ISelectProps } from '@components/Select';
+import { Select } from '@components/Select';
+import type { FC } from 'react';
+import React from 'react';
+
+export interface ISelectFieldProps extends Omit<IInputWrapperProps, 'htmlFor'> {
+  selectProps: Omit<ISelectProps, 'disabled' | 'children' | 'leadingTextWidth'>;
+}
+
+export const SelectField: FC<ISelectFieldProps> = ({
+  disabled = false,
+  selectProps,
+  children,
+  ...rest
```

Comment:
Sometimes we spread props and sometimes we don't. Do we have clear rules or something to decide when we do?

Reply:
I think it's fine to use the spread operator since we know what those props will be using typescript. I think the distinction is more around when you extend our prop types to all of the available props for a specific element which depends on the use case. In the case of input elements I think we don't want to restrict anything since our input components should be a thin wrapper.